apps/opik-backend/src/main/java/com/comet/opik/domain/DatasetItemDAO.java

```diff
@@ -137,7 +138,7 @@ INSERT INTO dataset_items (
             SELECT
                 id,
                 dataset_id,
-                <if(truncate)> mapApply((k, v) -> (k, replaceRegexpAll(v, '<truncate>', '"[image]"')), data) as data <else> data <endif>,
+                <if(truncate)> mapApply((k, v) -> (k, substring(replaceRegexpAll(v, '<truncate>', '"[image]"'), 1, <truncationSize>)), data) as data <else> data <endif>,
```

Comment:
not ao blocker, in followup PR catch 1 test that truncates and input large data above config, fetch and make sure truncation is applied.